decentraland/unity-renderer
# fix: camera turning off before teleport

## What does this PR change?

Fixes #2972 

The state of the camera is no longer dependent on the `rendererState `value. It now only depends on the state of the `LoadingHUD` and any `FullScreenHUD`.

## How to test the changes?

1. Load into the game as a guest or via a wallet.
2. Teleport to any field by using the Map ('M' button) or a chat command ('/goto random' or '/goto X,Y')
3. Check that there is no empty camera (terrain flash) before the LoadingUI started to fade in
4. Check that the camera is on when the LoadingUI start to fade out

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
